Products/PloneKeywordManager/browser/prefs_keywords_view.pt

```diff
@@ -177,11 +177,11 @@
                                  onclick string:document.forms['keyword_edit_form'].changeto.value='${subject}';; return true;;" />
           <label tal:content="subject" tal:attributes="for string:subject-${subject_id};">Subject</label>
           <tal:p5 tal:condition="view/is_plone_5">
-            <a href="#" tal:attributes="href string:${portal_url}/@@search?Subject=${subject}">
+            <a href="#" tal:attributes="href string:${portal_url}/@@search?${field}=${subject}" target="_blank">
```

Comment:
usually you want to look quickly, which articles are affected by this subject. so we found it useful to open this quick review in a new window.